src/shared/utils/utils.tsx

```diff
@@ -108,3 +110,8 @@ export function keysToLowerCase<T>(o: { [k: string]: T } = {}): {
     Object.entries(o).map(([k, v]) => [k.toLowerCase(), v])
   );
 }
+
+export const excludeKeys = (o: Record<Key, unknown>, omit: Key[]) => {
+  const setOmit = new Set(omit);
+  return Object.fromEntries(Object.entries(o).filter(([k]) => !setOmit.has(k)));
+};
```

Comment:
We could also use lodash's omit to do this, and it would have the same API as your code here.
Was this on purpose to avoid using a library? I'd say we already use that one throughout the code so it's not like we'll want to remove it anytime soon, and also its types are quite good.

Reply:
I thought we wanted to move to implementing simple utils for ourselves in the case we want to do away with lodash one day. I see lodash's `omit` used once in the codebase so I could easily replace with my `excludeKeys` or I could just remove my function - happy with either.

Reply:
Yep, if that's made on purpose it's fine, and the logic is fairly encapsulated so that's alright.

Reply:
Replaced the other instance of `omit` with the new util fn too.